Creiwry/irl-meeting-app
# UI: create views for users : login and signup

### Ticket Link
https://templatethpprojetfinal.atlassian.net/browse/THPPROJ-21
https://templatethpprojetfinal.atlassian.net/browse/THPPROJ-22
https://templatethpprojetfinal.atlassian.net/browse/THPPROJ-25

### User Story
Create sign up view (form helper and registrations#new)
Style sign up view (front-end of registrations#new)
Style front of sessions#new

### What does this PR do?
1. Landing page modified with button 'log out' in views/pages/welcome to be able to run many login/signup.
2. Links from devise/shared/_links modified with css class:'clkbtn' to be displayed in better way.
3. View for Signup created with css (views/devise/registrations/new)
4. View for Login created with css (views/devise/sessions/new)

### Where should the reviewer start?
From landing page

### How should this be manually tested?
1. Try to sign up / Log out / Log in
2. Check in Rails Console the value for 'User.last'

### Screenshots (if appropriate)
![image](https://github.com/Creiwry/irl-meeting-app/assets/137895449/e410da67-e16c-4283-8d4e-6ce00b89f33c)
![image](https://github.com/Creiwry/irl-meeting-app/assets/137895449/c22f2cb4-f617-425f-b352-212a21830df9)


### Are there any breaking changes?
should not be
